tests/conftest.py

```diff
@@ -11,9 +11,17 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Default parameters, environment variables, fixtures, and common routines for
+"""Default parameters, environment variables, fixtures, and shared objects
 the unit tests and manual checks."""
-
+import os
 import pytest
-import numpy as np
 import flamingpy
+
+# defaults
+TOL = 1e-5
+
+
+@pytest.fixture(scope="session")
+def tol():
+    """Numerical tolerance for equality tests."""
+    return float(os.environ.get("TOL", TOL))
```

Comment:
Is this future-proofing (I don't think it's currently used anywhere)?

Reply:
Yes, exactly, future-proofing.